.github/workflows/merge-next-notify.yml

```diff
@@ -1,11 +1,11 @@
 name: "Merge Next Notifier"
 
 on:
-  pull_request_target:
+  pull_request:
     types: 
       - labeled
     branches:
-      - next
+      - main
 
 jobs:
   check-label:
```

Comment:
try adding a GitHub secret. 

- name: Write Sticky Comment
        uses: marocchino/sticky-pull-request-comment@v2.2.0
        with:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          comment: |
            Hello! This is a sticky comment that will be added to the top of the pull request and will stay there until it is manually removed.